client/src/main/java/org/apache/rocketmq/client/impl/MQClientManager.java

```diff
@@ -53,8 +53,9 @@ public MQClientInstance getAndCreateMQClientInstance(final ClientConfig clientCo
             MQClientInstance prev = this.factoryTable.putIfAbsent(clientId, instance);
             if (prev != null) {
                 instance = prev;
-            } else {
                 log.warn("Previous MQClientInstance has created for clientId:[{}]", clientId);
+            } else {
+                log.info("new MQClientInstance has created for clientId:[{}]", clientId);
```

Comment:
It will be best if using active tune here.  Something like, Returned previous MQClientInstance for clientId: ..., Created new MQClientInstance for clientId: ...